### DISCUSSION TITLE
Suggested edit for documentation

### DISCUSSION THREAD

**Discussion by Sbrjt** [twbs/bootstrap#39382] [Ideas]
I think the docs about [embeds](https://getbootstrap.com/docs/4.0/utilities/embed/) should add a little note that [embedded YT video doesn't work on local server](https://stackoverflow.com/questions/51969269/).

**Comment by julien-deramond** [comment]
Thanks for the suggestion @Sbrjt 
I tried to run our documentation locally and went to http://localhost:9001/docs/5.3/helpers/ratio/#example (the equivalent of https://getbootstrap.com/docs/5.3/helpers/ratio/#example online) and the embedded YouTube works well.
For other use cases, I'd say that this YT video is only an example to demonstrate the ratio helper and so that any technical issues around `<iframe>`s, `<embed>`s, `<video>`s, and `<object>`s would be on the developer hand based on what's done in the website.
